Provide more detailed example for BLOM user_nl_blom (#362)
NorESMhub/NorESM: doc/configurations/experiment_environment.rst

  set BDMC2 = .15
  set NIWGF = .5
  set SRF_ANO3 = 0, 2, 2

> you need to include **set** before the name of the variable and it does not matter what namelist group the variable belong. To enable this handling of namelist parameters in BLOM, the namelist parameters require to be spelled in capital in both, user_nl_blom and cime_config/buildnml (the latter for developers, only).

Reviewer: ```suggestion
you need to include **set** before the name of the variable and it does not matter what namelist group the variable belong. Input entries in user_nl_blom are case-insensitive.
```

The input to the `user_nl_blom` file should not be case sensitive. The `buildnml` script capitalize all input variables before evaluating input from `user_nl_blom` (see the section "Create resolved namelist" inside the `cime_config/buildnml` file). This means that entries in the `buildnml` file must be capitalized if they should be available for change from `user_nl_blom`. I'm not sure this last point should be included here, since it is mostly relevant for model developers, and not for regular users of the model.

Author: Hi @TomasTorsvik , thanks for re-checking. Maybe one can simply write

Input entries in user_nl_blom are case-insensitive (for model developers: in the BLOM `buildnml`, they require to be capital though).

But I leave this to you.

Reviewer: Yes, I guess we might as well include it in this way.
